src/main/java/org/broadinstitute/hellbender/tools/BwaMemIndexImageCreator.java

```diff
@@ -19,25 +23,51 @@
 )
 public final class BwaMemIndexImageCreator extends CommandLineProgram {
 
+    public static final String USE_EXISTING_INDEX_FULL_NAME = "useExistingIndexFilesIfAvailable";
+    public static final String USE_EXISTING_INDEX_SHORT_NAME = "useIndexFiles";
+
     @Argument(fullName = StandardArgumentDefinitions.INPUT_LONG_NAME,
-              shortName = StandardArgumentDefinitions.INPUT_SHORT_NAME,
-              doc = "Input reference fasta file. The five bwa index files are assumed living in the same directory with the same prefix.")
+            shortName = StandardArgumentDefinitions.INPUT_SHORT_NAME,
+            doc = "Input reference fasta file. The five bwa index files are assumed living in the same directory with the same prefix.")
```

Comment:
I think the second sentence needs to be updated to something like: "The index image will be built directly from the reference FASTA, unless " + USE_EXISTING_INDEX_FULL_NAME + " is true."